contrib/get_supported_version_csi-sidecar.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import datetime
+import re
+from collections import defaultdict
+import subprocess
+import shutil
+from dateutil.relativedelta import relativedelta
```

Comment:
If this is an external dependency I think it needs to have a `pip install` command somewhere but it's a dev only tool so I think it's good.

Also consider that this file is going to be synced to all the CSI repos.

Reply:
`collections` is a built-in ; `dateutil` is an external dependency.

Next time I work on it I will replace it with just built-in ;-)